example/spec/parallel/discoverGranules/DiscoverGranulesDuplicateHandlingSpec.js

```diff
@@ -0,0 +1,178 @@
+'use strict';
```

Comment:
This test is running an ingest workflow similar to the serial Http spec test.    It should be safe in it's current configuration to run in parallel,  *however* I want to update it to use a different granule set as we're not deleting the granules when we clean up the other test, or this one for that matter.    Ticket forthcoming on that problem, as updating that host is proving problematic.

Reply:
The issue is *not* cleaning up granules.  It means the system isn't in a static state - and in this case, this test would collide with the existing test if there were further tests ingesting with it.    We really need a collection_002 with separate granule files on the server. 

I suppose I could configure S3 instead.... 🤔